specs/windows/wmi_tpm_info.table

```diff
@@ -0,0 +1,19 @@
+table_name("wmi_tpm_info")
+description("A table that lists the TPM related information.")
+schema([
+  Column("is_activated_initialvalue", TEXT, "Name of the Bios setting"),

```

Comment:
I am unsure why the WMI result contains the `InitialValue` suffix. Is it reporting the TPM's activated, enabled, owned state or something different?

Either way can we simplify these columns to be named `activated`, `enabled`, and `owned`? And can we have them be `INTEGER` types and use 0 for False and 1 for True.